Packages/com.unity.inputsystem/InputSystem/Editor/ProjectWideActions/ProjectWideActionsAsset.cs

```diff
@@ -102,6 +107,42 @@ private static InputActionAsset CreateNewActionAsset()
 
             return asset;
         }
+
+        // Writes the current project wide actions (`InputSystem.actions`) content to a file
+        // the Roslyn source generator will be able to read. This file is automatically generated
+        // and is created in the root of the User's Assets directory.
+        internal static void CreateRoslynAdditionalFile()
+        {
+            try
+            {
+                if (!Directory.Exists("Assets"))
+                    Directory.CreateDirectory("Assets");
+
+                var assetJson = InputSystem.actions.ToJson();
```

Comment:
Can we search for existing file in the AssetDB at any path, use that location if it exists and overwrite, then only create at the fixed path if not present elsewhere in the Assets folder.

This would allow people to move it.

Reply:
If we do this the delete would have to also search for all files of that name which does add complexity.

Reply:
For pre-2 this could could be sufficient so I wouldn't block merging for it.
Perhaps capture this additional ask in a task if you agree.